specification/communication/data-plane/Rooms/preview/2022-02-01-preview/communicationservicesrooms.json

```diff
@@ -0,0 +1,707 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Room Service",
+    "description": "Azure Communication Room Service",
+    "version": "2022-02-01"
+  },
+  "paths": {
+    "/rooms": {
+      "post": {
+        "tags": [
+          "Rooms"
+        ],
+        "summary": "Creates a new room.",
+        "operationId": "Rooms_CreateRoom",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "header",
+            "name": "Repeatability-Request-ID",
+            "description": "If specified, the client directs that the request is repeatable; that is, that the client can make the request multiple times with the same Repeatability-Request-Id and get back an appropriate response without the server executing the request multiple times. The value of the Repeatability-Request-Id is an opaque string representing a client-generated, globally unique for all time, identifier for the request. It is recommended to use version 4 (random) UUIDs.",
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "in": "header",
+            "name": "Repeatability-First-Sent",
+            "description": "If Repeatability-Request-ID header is specified, then Repeatability-First-Sent header must also be specified. The value should be the date and time at which the request was first created, expressed using the IMF-fixdate form of HTTP-date",
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "in": "body",
+            "name": "createRoomRequest",
+            "description": "The create room request body.",
+            "schema": {
+              "$ref": "#/definitions/CreateRoomRequest"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Request successful. The action returns a new `Room`.",
+            "schema": {
+              "$ref": "#/definitions/RoomModel"
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
```

Comment:
Do other teams document the `MS-CV` in the response headers and do we expose it in the server SDKs? Rooms service does return it in all our responses.